audit/audit.go

```diff
@@ -28,14 +29,26 @@ import (
 	"strings"
 )
 
-func ProcessPaths(libPaths []string, libs []string, files []string) (count int) {
+type Audit struct {
+	OssiConfig         config.OSSIConfig
+	HasProperOSSICreds bool
+}
+
+func New(ossiConfig config.OSSIConfig) *Audit {
+    return &Audit{
+        OssiConfig:         ossiConfig,
+        HasProperOSSICreds: len(ossiConfig.Username) > 0 && len(ossiConfig.Token) > 0,
```

Comment:
This is really cool, but you could LIKELY do it as a method hanging off Audit itself

Reply:
this was an artifact of testing as well :(